autoPyTorch/evaluation/abstract_evaluator.py

```diff
@@ -422,6 +516,15 @@ def _loss(self, y_true: np.ndarray, y_hat: np.ndarray) -> Dict[str, float]:
         The calculate_loss internally translate a score function to
         a minimization problem
 
+        Args:
+            y_true (np.ndarray):
+                The expect labels given by the original dataset
+            y_hat (np.ndarray):
```

Comment:
[scikit](https://scikit-learn.org/stable/modules/model_evaluation.html) is using `y_pred` instead of `y_hat`.

Reply:
I would like to honor the title of the PR and only address documentation-related changes. This looks like a refactor.